src/panels/ActionsPanel/_actions-panel.scss

```diff
@@ -27,4 +27,19 @@
     position: sticky;
     text-align: right;
   }
+
+  &__confirmation-modal {
+    &-sub-header {
+      color: $color-mid-dark;
+      font-size: 0.875rem;
```

Comment:
`@extend %small-text`

Reply:
This doesn't appear to work.
```
./src/panels/ActionsPanel/_actions-panel.scss (./node_modules/css-loader/dist/cjs.js??ref--5-oneOf-6-1!./node_modules/postcss-loader/src??postcss!./node_modules/resolve-url-loader??ref--5-oneOf-6-3!./node_modules/sass-loader/dist/cjs.js??ref--5-oneOf-6-4!./src/panels/ActionsPanel/_actions-panel.scss)
SassError: The target selector was not found.
Use "@extend %small-text !optional" to avoid this error.
   ╷
33 │       @extend %small-text;
   │       ^^^^^^^^^^^^^^^^^^^
   ╵
  src/panels/ActionsPanel/_actions-panel.scss 33:7  root stylesheet
```